tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -16,8 +16,6 @@
 
 class EmailHandler(NotificationPlugin):
 
-    handler = "smtp"
```

Comment:
why is this removed any specific reason??
How would you otherwise know the list of supported notifying methods... or anything of such sort